### PR TITLE
change default deploy type to deployToStagingSlotAndSwap

### PR DESCRIPTION
**Pull request by pasqualedevita**

### CODE REVIEW

**Comment by pasqualedevita** [comment]
deploy with swap seems ok @gunzip after merge this pr can you create a new minor version `7.9.4`?

File: azure-pipelines.yml [hunk 1]
```diff
@@ -70,7 +70,7 @@ parameters:
   - name: 'PRODUCTION_DEPLOY_TYPE'
     displayName: 'Method to achieve deployment in Production (if enabled):'
     type: string
-    default: deployToStagingSlot
```
Comment (gunzip) [inline azure-pipelines.yml:73]: can we hardcode this since it is not supported on linux ?

**Comment by gunzip** [comment]
I suggest to remove all the 'deployng to test' part

File: azure-pipelines.yml (hunk 1 above)
Comment (pasqualedevita) [reply azure-pipelines.yml:73]: On linux is not supported autoswap option, but we can deploy on staging slot.
I think that we can maintain this option so if we need to test something on staging we can do it.

**Comment by pasqualedevita** [comment]
> I suggest to remove all the 'deployng to test' part

Tomorrow I'll porting this pipeline to new standard such as https://github.com/pagopa/io-developer-portal-backend/pull/116
cc @balanza

**Pull request merged**